.github/workflows/CI.yml

```diff
@@ -15,6 +15,13 @@ jobs:
         uses: actions/setup-java@v1
         with:
           java-version: 1.11
+      - name: Sync readme
```

Comment:
It should run only on CD step but not CI so please move it out from here to CD.yaml. 
Also please add the same for the second image we deploy - home-data-migration.

Reply:
Moved to CD and added action for home-data-migration